indico/web/flask/app.py

```diff
@@ -238,6 +238,7 @@ def setup_jinja(app):
     app.add_template_filter(date_time_util.format_pretty_date)
     app.add_template_filter(date_time_util.format_pretty_datetime)
     app.add_template_filter(date_time_util.server_to_utc)
+    app.add_template_filter(date_time_util.convert_py_weekdays_to_js)
```

Comment:
Just make this a property on the field class. I don't expect this to be used in (m)any other places...